gcc-cross/build/patches/cross-tool-ng/cross-tool-ng-1.22.0.patch

```diff
@@ -0,0 +1,21 @@
+commit 9f83a56a3a3354fd05d86e444014a9279dd73d5e
+Author: Daniel Black <daniel@linux.ibm.com>
```

Comment:
Thanks!

Reply:
easy. note i still had the following fail in 1.22.0:
<pre>
f8eacdbfe606        mattgodbolt/gcc-cross:latest   "./build.sh arm 7.1.0"   4 days ago          Exited (2) 4 days ago                       arm-7.1.0
6177f2385749        mattgodbolt/gcc-cross:latest   "./build.sh arm64 7.…"   4 days ago          Exited (2) 4 days ago                       arm64-7.1.0
2d5984d6619e        mattgodbolt/gcc-cross:latest   "./build.sh arm64 6.…"   4 days ago          Exited (2) 4 days ago                       arm64-6.3.0
d2935a746c58        mattgodbolt/gcc-cross:latest   "./build.sh arm 6.3.0"   4 days ago          Exited (2) 4 days ago                       arm-6.3.0
</pre>

With errors like:
<pre>
[EXTRA]    Configuring core C gcc compiler
[EXTRA]    Building gcc
[ERROR]    /opt/.build/src/gcc-6.3.0/gcc/ubsan.c:1474:23: error: ISO C++ forbids comparison between pointer and integer [-fpermissive]
[ERROR]    make[3]: *** [ubsan.o] Error 1
[ERROR]    make[3]: *** Waiting for unfinished jobs....
[ERROR]    make[2]: *** [all-gcc] Error 2
</pre>

Which is the error: https://github.com/crosstool-ng/crosstool-ng/issues/992

As these are all arm ones run a minor version by @dbroemmel 's contribution I haven't bothered migrating these to the ct-ng 1.22.0+ version.